packages/mantine/src/components/collection/Collection.tsx

```diff
@@ -190,9 +212,17 @@ export const Collection = <T,>(props: CollectionProps<T>) => {
                         {...others}
                     >
                         <Stack spacing={spacing}>
-                            {items}
-                            {provided.placeholder}
-                            {_addButton}
+                            <Stack spacing={'xs'}>
+                                <Title order={3}>{label}</Title>
+                                <Title order={6} color="dimmed" weight={200}>
+                                    {description}
+                                </Title>
+                            </Stack>
```

Comment:
We probably should be using the exact same style as the other input components. You can follow the CodeEditor as an example https://github.com/coveo/plasma/blob/master/packages/mantine/src/components/code-editor/CodeEditor.tsx#L129-L147

Reply:
noted